samples/rabbitmq/native-integration/sample.md

```diff
@@ -10,12 +11,12 @@ related:
 
 ## Code walk-through
 
-The sample consists of 2 console apps, a simple NServiceBus receiver and a native sender. These apps demonstrate how to have senders on other platforms send messages to NServiceBus endpoints. While the .NET RabbitMQ client is used in this sample, the approach demonstrated can be used with any of the [official](https://www.rabbitmq.com/download.html) or [community](https://www.rabbitmq.com/devtools.html) native RabbitMQ client libraries.
+The sample consists of two endpoints, a simple NServiceBus receiver and a native sender. These endpoints demonstrate how to have senders on other platforms send messages to NServiceBus endpoints. While the .NET RabbitMQ client is used in this sample, the approach demonstrated can be used with any of the [official](https://www.rabbitmq.com/download.html) or [community](https://www.rabbitmq.com/devtools.html) native RabbitMQ client libraries.
```

Comment:
Is endpoint the right word here? One of the projects doesn't use NServiceBus at all, only the native RabbitMQ client. Seems like endpoint has a specific meaning that wouldn't apply to that project.